finalise.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env sh
```

Comment:
Can you add the ACCESS-NRI copyright header / licence ?

Reply:
Added in this f77f4f2